tests/test_func.py

```diff
@@ -20,3 +22,13 @@ def test_set_returning_func(db: gp.Database):
     generate_series = gp.function("generate_series", db)
     results = generate_series(0, 9, as_name="id").to_table().fetch()
     assert sorted([row["id"] for row in results]) == list(range(10))
+
+
+def test_create_func(db: gp.Database):
+    @gp.create_function(db)
+    def add(a: int, b: int) -> int:
+        return a + b
+
+    for row in add(1, 2, as_name="sum").to_table().fetch():
+        assert row["sum"] == 1 + 2
+        assert row["sum"] == inspect.unwrap(add)(1, 2)
```

Comment:
Add test `create_function()` with no param type in the signature.
```
@gp.create_function(db)
    def add(a, b) -> int:
        return a + b
```

Test with multi-lines function body to make sure the indention is correct

```
@gp.create_function(db)
    def add(a, b) -> int:
        c = a + b
        return c
```

Test with space/tab indention function body 


`# fmt: on/off` may help if you want to turn the black format off.

Reply:
Currently, the `db` param is required.

The indention issue is fixed.

Reply:
Added a test case for tab-indented function definition

Added a TODO for testing

Reply:
create_function() with no param type currently does NOT work.

Reply:
Maybe we can add an optional `signature` argument to allow the user specify the signature explictly.